```
./utils/gyb_syntax_support/Utils.py:3:1: E302 expected 2 blank lines, found 1
./utils/gyb_syntax_support/Node.py:1:1: F401 'sys' imported but unused
./utils/gyb_syntax_support/Node.py:5:1: I100 Import statements are in the wrong order. 'from .Utils import error' should be before 'from .kinds import SYNTAX_BASE_KINDS, kind_to_type, lowercase_first_word'
./utils/gyb_syntax_support/Node.py:5:1: I202 Additional newline in a group of imports. 'from .Utils import error' is identified as Application Relative and 'from .kinds import SYNTAX_BASE_KINDS, kind_to_type, lowercase_first_word' is identified as Application Relative.
./utils/gyb_syntax_support/Node.py:9:1: E303 too many blank lines (3)
```

--- a/lib/Parse/ParsedSyntaxBuilders.cpp.gyb
+++ b/lib/Parse/ParsedSyntaxBuilders.cpp.gyb
@@ -85,6 +85,7 @@ void Parsed${node.name}Builder::finishLayout(bool deferred) {
 % if node.children:
 %   for (idx, child) in enumerate(node.children):
 %     child_elt = None
+%     child_elt_name = child.name + 'Member'
 %     child_node = NODE_MAP.get(child.syntax_kind)
 %     if child_node and child_node.is_syntax_collection():
 %       child_elt = child_node.collection_element_name

--- a/utils/gyb_syntax_support/Classification.py
+++ b/utils/gyb_syntax_support/Classification.py
@@ -1,4 +1,4 @@
-from .Node import error
+from .Utils import error
 from .kinds import lowercase_first_word  # noqa: I201
 
 

--- a/utils/gyb_syntax_support/Node.py
+++ b/utils/gyb_syntax_support/Node.py
@@ -1,11 +1,5 @@
-import sys  # noqa: I201
-
+from .Utils import error
 from .kinds import SYNTAX_BASE_KINDS, kind_to_type, lowercase_first_word
-
-
-def error(msg):
-    print('error: ' + msg, file=sys.stderr)
-    sys.exit(-1)
 
 
 class Node(object):

--- a/utils/gyb_syntax_support/NodeSerializationCodes.py
+++ b/utils/gyb_syntax_support/NodeSerializationCodes.py
@@ -1,4 +1,4 @@
-from .Node import error
+from .Utils import error
 
 
 SYNTAX_NODE_SERIALIZATION_CODES = {

--- a/utils/gyb_syntax_support/Token.py
+++ b/utils/gyb_syntax_support/Token.py
@@ -1,5 +1,5 @@
 from .Classification import classification_by_name
-from .Node import error  # noqa: I201
+from .Utils import error  # noqa: I201
 from .kinds import lowercase_first_word  # noqa: I201
 
 

--- a/utils/gyb_syntax_support/Trivia.py
+++ b/utils/gyb_syntax_support/Trivia.py
@@ -1,4 +1,4 @@
-from .Node import error
+from .Utils import error
 from .kinds import lowercase_first_word  # noqa: I201
 
 

--- a/utils/gyb_syntax_support/Utils.py
+++ b/utils/gyb_syntax_support/Utils.py
@@ -1,0 +1,6 @@
+import sys
+
+
+def error(msg):
+    print('error: ' + msg, file=sys.stderr)
+    sys.exit(-1)

